### PR TITLE
3.2 version OHOS adaptation

### PR DESCRIPTION
**Pull request by IceBBBBB**

### CODE REVIEW

File: cocos/scripting/lua-bindings/manual/CCLuaEngine.cpp [hunk 1]
```diff
@@ -57,9 +57,9 @@ bool LuaEngine::init(void)
 {
     _stack = LuaStack::create();
     _stack->retain();
-    executeScriptFile("DeprecatedEnum.lua");
-    executeScriptFile("DeprecatedClass.lua");
-    executeScriptFile("Deprecated.lua");
+    executeScriptFile("cocos/DeprecatedEnum.lua");
+    executeScriptFile("cocos/DeprecatedClass.lua");
+    executeScriptFile("cocos/Deprecated.lua");
```
Comment (minggo) [inline cocos/scripting/lua-bindings/manual/CCLuaEngine.cpp]: Why change path?

File: tests/lua-empty-test/project/Classes/AppDelegate.cpp [hunk 2]
```diff
@@ -28,7 +28,8 @@ bool AppDelegate::applicationDidFinishLaunching()
     
     //The call was commented because it will lead to ZeroBrane Studio can't find correct context when debugging
     //engine->executeScriptFile("src/hello.lua");
-    engine->executeString("require 'src/hello.lua'");
+    FileUtils::getInstance()->addSearchPath("");
```
Comment (minggo) [inline tests/lua-empty-test/project/Classes/AppDelegate.cpp]: Why add an empty path?

File: tests/lua-tests/src/controller.lua [hunk 3]
```diff
@@ -66,14 +71,15 @@ local function addSearchPath(resPrefix, height)
     end
 
     fileUtils:setSearchPaths(searchPaths)
+    fileUtils:setSearchPaths("")
```
Comment (minggo) [inline tests/lua-tests/src/controller.lua]: Why add an empty path?

File: cocos/scripting/lua-bindings/manual/CCLuaEngine.cpp (hunk 1 above)
Comment (IceBBBBB) [reply cocos/scripting/lua-bindings/manual/CCLuaEngine.cpp]: modified

File: tests/lua-empty-test/project/Classes/AppDelegate.cpp (hunk 2 above)
Comment (IceBBBBB) [reply tests/lua-empty-test/project/Classes/AppDelegate.cpp]: modified

File: tests/lua-tests/src/controller.lua (hunk 3 above)
Comment (IceBBBBB) [reply tests/lua-tests/src/controller.lua]: modified

Comment (minggo) [reply tests/lua-tests/src/controller.lua]: It seems it is not modified.

File: tests/lua-empty-test/project/Classes/AppDelegate.cpp (hunk 2 above)
Comment (minggo) [reply tests/lua-empty-test/project/Classes/AppDelegate.cpp]: It seems it is not modified.

Comment (IceBBBBB) [reply tests/lua-empty-test/project/Classes/AppDelegate.cpp]: sorry, modified

File: tests/lua-tests/src/controller.lua (hunk 3 above)
Comment (IceBBBBB) [reply tests/lua-tests/src/controller.lua]: sorry, modified

**Pull request merged**